docs/_docs/versions.md

```diff
@@ -67,11 +67,23 @@ If you're working with an integration that's currently private, the _Promote_ op
 
 ![Deprecate Zapier integration version](https://cdn.zapier.com/storage/photos/dd6acdd75278ecd733a5f3945ea641a2.png)
 
-If you did introduce breaking changes to your integration and want existing users to switch to the new version, you can deprecate the old version. That will let your older integration and Zaps that use it continue to run—but they will include a `(Legacy)` tag in the Zapier editor to prompt users to re-build the Zap with the new integration. Additionally, when adding a new Zap step, Zapier will no longer show the older, deprecated version of your integration as an option.
+If you introduce breaking changes to your integration and want existing users to switch to the new version, you can deprecate the old version.
```

Comment:
This is good. I'd mention that deprecation is sort of a "last resort" for APIs that are going away or breaking changes on the partner side. Ideally, you can either migrate users forward or leave them safely on the old version if it continues to work. 

Forcing users to update their zaps (especially if they have a lot of them) is really disruptive.

Reply:
I can definitely add a note here about the reasons to do this or not.